File: ProjetoPOO/Models/Usuario.cs
using System;

namespace ProjetoPOO.Models
{
    public class Usuario : ModeloId
    {
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Senha { get; set; }
        public int NivelAcesso { get; set; }


        public Usuario() {}
        public Usuario(int id, string nome, string email, string senha, int nivel)
        {
            Id = id;
            Nome = nome;
            Email = email;
            Senha = senha;
            NivelAcesso = nivel;
        }

        public override string ToString()
        {
            return $"Id: {Id} | Nome: {Nome} | Email: {Email}";
        }
    }
}

File: ProjetoPOOPython/Models/usuario.py
from Models.modelo_id import ModeloId

class Usuario(ModeloId):
    def __init__(self, id=0, nome="", email="", senha="", nivel=0):
        super().__init__(id)
        self.Nome = nome
        self.Email = email
        self.Senha = senha
        self.NivelAcesso = nivel

    def to_dict(self):
        return {
            "Id": self.id,
            "Nome": self.Nome,
            "Email": self.Email,
            "Senha": self.Senha,
            "NivelAcesso": self.NivelAcesso
        }